roles/os_must_gather/tasks/main.yml

```diff
@@ -72,7 +72,13 @@
     - name: Get exact must-gather output folder name
       ansible.builtin.find:
         paths: "{{ cifmw_os_must_gather_output_dir }}/logs"
-        patterns: "quay-io-openstack-k8s-operators-openstack-must-gather*"
+        patterns: >-
```

Comment:
You'll hate me, buuut...

can we at least get a TODO to get some molecule coverage there, like we have for the cifmw_job_uri parsing in the reproducer role?

Also, a couple of example input/output as a leading comment here would be really, really appreciated.

Reply:
done!